src/display/editor/ink.js

```diff
@@ -515,6 +530,11 @@ class InkEditor extends AnnotationEditor {
 
     super.render();
     this.div.classList.add("editing");
+
+    InkEditor._l10nPromise
+      .get("editor_ink_aria_label")
+      .then(msg => this.div.setAttribute("aria-label", msg));
```

Comment:
Given the asynchronicity of the code, would it be safer to do the following?
```suggestion
      .then(msg => this.div?.setAttribute("aria-label", msg));
```